core/domain/exp_jobs_one_off_test.py

```diff
@@ -1466,10 +1478,9 @@ def test_for_migration_job(self):
                 'content_id': 'default_outcome',
                 'html': (
                     '<pre>Hello this is <b> testing '
-                    '<oppia-noninteractive-image filepath-with-value="amp;quot;'
-                    'random.png&amp;quot;"></oppia-noninteractive-image> in '
-                    '</b>progress</pre>'
-
+                    '<oppia-noninteractive-image filepath-with-value='
+                    '"&amp;quot;random.png&amp;quot;">'
```

Comment:
@bansalnitish do you know why the change here was not caught before?

Reply:
Hi @seanlip, it was not caught since in RTE format validation, we are only extracting tag names and their parents to check valid relations and we don't check the content. The content is only checked inside collapsible and tabs to check the tags and parents inside them.